Octokit.Tests/Helpers/Arg.cs

```diff
@@ -72,5 +72,10 @@ public static ApiOptions ApiOptions
         {
             get { return Arg.Any<ApiOptions>(); }
         }
+
+        public static string AnyAcceptHeaders
+        {
+            get { return Arg.Any<string>(); }
```

Comment:
I'd rather be explicit about exactly what Accept Headers are being validated in unit tests, rather than saying "any accept header" is fine.  So can we remove this and be explicit in all the tests?

Reply:
I introduced this specifically for the `IsALukeWarmObservable` tests, because these tests already does not care about the actual URI. I think it makes this test more readable and maintainable.

If Accept headers are correct is tested in different set of tests which are specialized for this.

Reply:
Yep fair enough to leave the "any" in place for these tests since they don't aim to test the actual endpoint

Reply:
Ok, I will use AnyAcceptHeaders in observable tests.